.github/workflows/ci.yml

```diff
@@ -234,7 +234,7 @@ jobs:
           find /usr/local/bin -type l -exec sh -c 'readlink -f "$1" \
           | grep -q ^/Library/Frameworks/Python.framework/Versions/' _ {} \; -exec rm -v {} \;
           HOMEBREW_NO_INSTALLED_DEPENDENTS_CHECK=1 \
-          brew install dpkg findutils gnu-tar llvm pkg-config qemu
+          brew install dpkg findutils gnu-tar llvm pkg-config libelf qemu
```

Comment:
we should remove this commit?

Reply:
afaict we still need it - it was an implicit dependency of libbpf before but we didn't install it.
once macOS CI is green again (which i'm hoping my last commit should do) i'll do an experiment to see whether it is still required.

Reply:
please drop a note here when you learn the answer

Reply:
It logs a warning :)